Explorer/Assets/DCL/Landscape/TerrainGenerator.cs

```diff
@@ -933,104 +1018,77 @@ private bool OverlapsOccupiedParcel(float2 position, float radius)
 
         private static string treeFilePath => $"{Application.streamingAssetsPath}/Trees.bin";
 
-        private static void SaveTrees(ChunkModel[] chunks)
+        private static void SaveTrees(ChunkModel[] chunks, TerrainGenerationData terrainData)
```

Comment:
Optional: move this and dependent code to editor assembly and turn it into a tool.